massa-node/src/main.rs

```diff
@@ -318,9 +326,13 @@ async fn launch(
         Err(err) => panic!("critical error detected in the bootstrap process: {}", err),
     };
 
+    if !final_state.read().is_db_valid() {
+        // TODO: Bootstrap again instead of panicking
```

Comment:
Same, be sure to create an issue to track this

Reply:
I grouped both TODOs in the follow-up issue: #4000